toolset/benchmark/framework_test.py

```diff
@@ -289,12 +292,25 @@ def verify_type(test_type):
       test.passed = all(result is 'pass' for (result, reason, url) in results)
       
       def output_result(result, reason, url):
-        out.write("   %s for %s\n" % (result.upper(), url))
-        print "   %s for %s" % (result.upper(), url)
+        specific_rules_url = "http://frameworkbenchmarks.readthedocs.org/en/latest/Project-Information/Framework-Tests/#specific-test-requirements"
+        color = Fore.GREEN
+        showRules = False
+        if result.upper() == "WARN":
+          color = Fore.YELLOW
+          showRules = True
+        elif result.upper() == "FAIL":
+          color = Fore.RED
+          showRules = True
+
+        out.write(("   " + color + "%s" + Style.RESET_ALL + " for %s\n") % (result.upper(), url))
+        print ("   " + color + "%s" + Style.RESET_ALL + " for %s") % (result.upper(), url)
         if reason is not None and len(reason) != 0:
           for line in reason.splitlines():
             out.write("     " + line + '\n')
             print "     " + line
+          if showRules:
+            out.write("     See %s" % specific_rules_url);
+            print "     See %s" % specific_rules_url
 
       [output_result(r1,r2,url) for (r1, r2, url) in results]
```

Comment:
How about replacing the `showRules` lines above with a single conditional underneath line 315, such as this: 

```
if not test.passed:
  out.write("     See %s" % specific_rules_url)
  print "     See %s" % specific_rules_url
```

So that the rules URL is only printed a single time at the end instead of once per URL tested